From 570ce374be3516da506baf078de153dfa2c548e1 Mon Sep 17 00:00:00 2001
From: xbw <xbw@rock-chips.com>
Date: Sun, 27 Jan 2013 17:36:17 +0800
Subject: [PATCH] IO voltage: This voltage value must be set in accordance with
 the actual situation.   I am sorry annotation errors in the commit
 fd825f216270d012f2e30ab3ceb179428237ec68

---
 arch/arm/mach-rk30/common.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-rk30/common.c b/arch/arm/mach-rk30/common.c
index 4c08dfa51141..6f2d7a1db6b3 100755
--- a/arch/arm/mach-rk30/common.c
+++ b/arch/arm/mach-rk30/common.c
@@ -42,9 +42,10 @@ static void __init rk30_cpu_axi_init(void)
 
 static void __init rk30_io_drive_strength_init(void)
 {
-//modifyed by xbw£»at 2013-01-26
-//This value must be set in accordance with the actual situation.
+//modifyed  by xbw£»at 2013-01-26
+//This voltage value must be set in accordance with the actual situation.
 //so set the value in each driver their own file.
+
 #if 0 //#if defined(CONFIG_ARCH_RK3066B) || defined(CONFIG_ARCH_RK3188)
 	writel_relaxed(0x11001100, RK30_GRF_BASE + GRF_IO_CON4);
 #endif
-- 
2.35.3

